src/utils/getApplicableFormats.ts

```diff
@@ -3,7 +3,18 @@ import { RegularNode, SchemaNodeKind } from '@stoplight/json-schema-tree';
 import { COMMON_JSON_SCHEMA_AND_OAS_FORMATS } from '../consts';
 
 export function getApplicableFormats(schemaNode: RegularNode): [type: SchemaNodeKind, format: string] | null {
+  // JSON Schema itself doesn't directly support defining binary data types.
```

Comment:
can we improve this documentation to say that this isn't a json schema feature but an open api one?

Reply:
I have addressed your comment.